app/src/test/java/org/simple/clinic/registration/facility/RegistrationFacilitySelectionScreenControllerTest.kt

```diff
@@ -139,21 +143,27 @@ class RegistrationFacilitySelectionScreenControllerTest {
 
   @Test
   fun `when screen is started then location should only be read once`() {
```

Comment:
The test for checking if stale location has been discarded seems to be missing. Can you please add this?

Reply:
Good catch! I've added a test.

Reply:
What do you think?

https://github.com/simpledotorg/simple-android/blob/6055b777dc2ec9d372c2c2a010e3812c1ec66e5f/app/src/test/java/org/simple/clinic/registration/facility/RegistrationFacilitySelectionScreenControllerTest.kt#L172